.github/workflows/docker-push.yml

```diff
@@ -0,0 +1,68 @@
+---
+name: "Dockerfile Push"
+on:  # yamllint disable-line rule:truthy
+  push:
+    paths:
+      - .devcontainer/Dockerfile
+      - .github/workflows/docker-push.yml
+      - lte/gateway/docker/mme/Dockerfile.ubuntu20.04
+  schedule:
+    # Run four times a day (hours zero, six, twelve and eighteen)
+    - cron:  '0 0,6,12,18 * * *'
+
+jobs:
+  build_devcontainer_dockerfile:
+    env:
+      DOCKERFILE: .devcontainer/Dockerfile
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: Check Out Repo
+        uses: actions/checkout@v2
+      -
```

Comment:
indentation is a bit weird.
I personally use hyphen then definition, not hyphen line break and then something.
if we want to have this style it would be better to be the same in the entire file

Reply:
Yeah I think we have a mix across our workflow files. I'll move this file to hyphen and then def.

Reply:
Frankly I tend not to care about these things and just want consistency - love me some auto-formatters for that reason! Wonder if there is one for GH Workflow YAML files.. Must be an opinionated YAML formatter out there somewhere.